mautic/mautic
# [UX] Change "System Report?" to "Visible for other users"

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This update revises the label from “System Report?” to “Visible for Other Users” to improve clarity and intuitiveness. The new label directly conveys the functionality, allowing users to understand the option’s purpose without relying on additional tooltips. This change simplifies the user interface, making it more accessible and self-explanatory, thereby enhancing the user experience.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Reports
3. Create new
4. Look at the label

![image](https://github.com/mautic/mautic/assets/116097999/82b0f5eb-c699-4f0c-bcd3-18c7a289dc2e)
